net/ipv4/tcp_output.c

```diff
@@ -2540,6 +2676,20 @@ static int tcp_mtu_probe(struct sock *sk)
 	}
 	tcp_init_tso_segs(nskb, nskb->len);
 
+#ifdef CONFIG_SECURITY_TEMPESTA
+	if (!skb_tfw_tls_type(nskb) || WARN_ON_ONCE(!sk->sk_write_xmit))
+		goto transmit;
```

Comment:
This part is different from 5.10  and I didn't see this change in other patches for 6.12 and 6.8 - a missed patch in 5.10?